Sources/Deferred.swift

```diff
@@ -56,32 +62,27 @@ public struct Deferred<Value>: FutureType, PromiseType {
 
     // MARK: FutureType
 
-    private func upon(queue: dispatch_queue_t, options inOptions: dispatch_block_flags_t, body: Value -> Void) -> dispatch_block_t {
+    private func upon(executor: ExecutorType, options inOptions: dispatch_block_flags_t, body: Value -> Void) -> dispatch_block_t {
```

Comment:
With the new "executor" verbiage, "body" might make more sense as "execute body: …".